portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,28 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Mannie's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat|Prata&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
   <body>
+        <h1>Mannie's Portfolio</h1>
+        <ul class="nav">
+            <li><a class="active" href="index.html">Home</a></li>
+            <li><a href="travelpics.html">Gallery</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <img src="images/MANNIE.gif"/>
+      <p>This is my portfolio. I am a second year Computer Science and Engineer Major attending University of California, Merced.</p>
+      <p>I was born and raised in Southern California. </p>
+      <p>Some of my hobbies include playing tennis, going on hikes, and doodling. Recently I have enjoyed songwriting with my friends. </p>
+      <p>I also enjoy traveling! I have had the pleasure of visiting Canada, Brazil, Korea, Japan, and Thailand. The country I visit most comfortably is probably Thailand because I speak the language and have family there. I also do a lot of traveling within the states! Click below to see some pictures from my travels:</p>
+      <a href="travelpics.html">Travel Photo Gallery</a>
+      <p>One of my favorite shows is Brooklyn Nine-Nine. Click below to get a random quote from the show:</p>
+      <button onclick="addRandomQuote()"> Fav Brooklyn 99 Quotes</button>
```

Comment:
Consider adding your onclick listener in the Javascript file instead of putting it in an onclick property in the HTML file. Reference on how to do this:

https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Event_handlers

It's usually better practice to have the HTML simply have content, but not behavior. And all behavior can be driven by code in the JS files. This way, it's a lot easier to debug and find the cause of anything in your webpages: HTML files only contain content (no behavior and no styling), CSS files only contain styling (as little content as possible and as little behavior as possible), and JS files only contain behavior (as little content & styling as possible).

CSS and JS can kind of be hard to restrict entirely (since you must use JS to dynamically change styling, and some CSS selectors like :active are sort of a behavior/style hybrid), but that's overall the pattern I've seen in most good frontend code.